src/content/advanced/access-management/iam-roles-for-service-accounts/index.md

```diff
@@ -495,6 +495,15 @@ Modify the trust entity of your AWS IAM roles with the new identity provider URL
 
 For cross account roles please follow the [guide](#cross-account) above and replace the `CLOUDFRONT_DOMAIN` with the new `CLOUDFRONT_ALTERNATE_DOMAIN`.
 
+Login into the AWS on the account where the cluster is running:
+
+- Grab the URL of the Identity Provider in your current cluster `IAM > Identity Providers`. It will look like `https://irsa.CLUSTER_ID.k8s.INSTALLATION_NAME.BASE_DOMAIN`.
+
+Login into the account where the IAM role is located and create an Identity Provider in `IAM > Identity Providers`:
+
+- Set `Provider URL` to the previously gathered URL and click the `Get thumbprint` to import the certificate.
```

Comment:
Not sure if this is clear to customers. Should we add screenshots for the steps? Not really needed if this is a rare corner case.